swan-cern/scripts/10_spark.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Author: Danilo Piparo, Enric Tejedor, Pedro Maximino 2023
+# Copyright CERN
+# The environment for Spark is configured here.
+
+# Spark configuration
```

Comment:
Shall we describe what this file does? If not here, maybe put a but more comments explaining whenever we do smth.